packages/meditrak-server/src/devops/getIsProductionEnvironment.js

```diff
@@ -3,4 +3,5 @@
  * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
  */
 
-export const getIsProductionEnvironment = () => process.env.IS_PRODUCTION_ENVIRONMENT === 'true';
+export const getIsProductionEnvironment = () =>
+  process.env.IS_PRODUCTION_ENVIRONMENT === 'true' && !process.env.CI_BUILD_ID;
```

Comment:
Do we need to change this in `web-config-server/src/utils/getIsProductionEnvironment.js` too?

Reply:
We 100% do! Great catch :+1:

Reply:
Weird the web-config-server build passed on that build you linked... but it should have failed??? Maybe the env var isn't being set correctly for web-config-server container...